articles/cloud-storage/cs-how-install-cifs-ecs.md

```diff
@@ -0,0 +1,125 @@
+toc_mdlink: cs-how-install-cifs-ecs.md
+---
+
+# How to install CIFS-ECS
+
+## What is CIFS-ECS?
+
+CIFS-ECS is a free application that runs on Windows and Windows Server, exposing UKCloud Cloud Storage as file systems. It’s designed as an easy way to use object storage by enabling Windows applications to interface with Cloud Storage servers using the popular S3 REST API.
+
+If you require a fully-integrated cloud gateway appliance with features like advanced deduplication and location caching, you may want to consider [CloudArrary](cs-how-install-cloudarray.md) as an alternative. 
+
+    > [!NOTE]
+    > CloudArray is not a free application like CIFS-ECS.
+
+## Use cases
+
+You can use CIFS-ECS to:
+
+- Expose UKCloud Cloud Storage for native access on Windows through a centralised server With CIFS-ECS installed on a centralised Windows Server installation.
+
+- Present Cloud Storage as storage for applications, without additional development.
+
+- Use Cloud Storage as a target for backups or long-term file retention.
+
+## Before you begin
+
+CIFS-ECS offers a single caching option:
+
+**Stub**: files are initially stored locally and replaced with a stub, or shortcut, after a period of time. When you access a file that has been stubbed, it will be retrieved from the cloud. This offers the best scalability, as you can use a virtually unlimited amount of cloud storage.
+
+## Installing CIFS-ECS
+
+1. Navigate to the following Portal page to download the ZIP file that contains the CIFS-ECS installer: [CIFS-ECS Tool] (https://portal.skyscapecloud.com/support/knowledge_centre/85dc54ff-7d21-43ac-9cfa-88e87de7ba27)
+
+    To download CIFS-ECS you'll need your UKCloud Portal log in details.
+
+1.	Extract and run the setup launcher from the ZIP file.
+
+2.	Step through the wizard, adjusting values to suit your environment.
+
+    > [!NOTE]
+    > The Data Directory is used as a local cache for any content you place in the CIFS-ECS drive, so you may want to put this on a non-system volume.
+
+3.	When the installation is complete, click Add **CIFS-ECS Drive** to open the *Drive Parameters* page.
+
+1.  On the *Drive Parameters* page, select any available drive letter from the drop down menu to use as the CIFS-ECS drive then click **Next**.
+
+    Optional: give your ECS drive a description.
+
+    ![*Drive parameters* page](images/cs-ecs-installtion-step-one.png)
+
+5.	On the Settings page, click the Hosts button to configure the connection to the Cloud Storage service.
+
+6.	On the General tab, click the Add button and configure the settings as shown in the following example:
+
+    - You can find the User ID in the Storage section of the UKCloud Portal.
+
+    - If you’ve forgotten or don’t know your Secret Access Key, you can reset it in the Storage section of the Portal
+
+        > [!NOTE]
+        > The 'Server Name' and 'Description' can be anything, but must not be blank. In the example it is shown as 'UKCloud' and 'UKCloud - ECS Drive'
```

Comment:
Bold for Server Name and Description (remove the quotation marks)

For UKCloud and UKCloud - ECS Drive use backticks instead of quotation marks (`)